src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -975,6 +975,13 @@ namespace NServiceBus
     {
         public static NServiceBus.SecondLevelRetries.Config.SecondLevelRetriesSettings SecondLevelRetries(this NServiceBus.EndpointConfiguration config) { }
     }
+    public class SecondLevelRetryContext
+    {
+        public SecondLevelRetryContext() { }
+        public System.Exception Exception { get; set; }
+        public NServiceBus.Transports.IncomingMessage Message { get; set; }
+        public int SecondLevelRetryAttempt { get; set; }
```

Comment:
NumberOfAttemps? Not sure if repeating the word SLR provides any meaningful context